bin/sidecli.re

```diff
@@ -88,18 +88,17 @@ let uri = {
 };
 
 let address = {
-  open Tezos_interop.Key_hash;
   let parser = string =>
-    Tezos_interop.Key_hash.of_string(string)
-    |> Option.map((Ed25519(hash)) => Wallet.address_of_blake(hash))
+    Crypto.Ed25519.Key_hash.of_string(string)
+    |> Option.map(hash => Wallet.address_of_blake(hash))
```

Comment:
This is still a regression, address is not supposed to accept only ed25519. It was like that because ed25519 was the only supported platform.